.github/workflows/typedoc.yml

```
@@ -0,0 +1,54 @@
+        run: |
+          cd frontend
+          typedoc --out docs/api \
+            --entryPoints app components lib types \
+            --exclude '**/*.test.ts' '**/*.spec.ts' '**/node_modules/**' \
+            --theme default \
+            --name "Shadower Frontend API" \
+            --readme frontend/README.md
+        # Auto-generates TypeScript API docs agents can read
+
+      - name: Deploy docs to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.8)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)